webpack.config.prod.js

```diff
@@ -9,7 +9,8 @@ module.exports = {
   },
   output: {
     filename: '[name].js',
-    path: path.join(__dirname, 'packages/formation/dist')
+    path: path.join(__dirname, 'packages/formation/dist'),
+    publicPath: './'
```

Comment:
We can use `publicPath` to set the output files to use a relative path for their urls.